converter/src/test/java/gov/cms/qpp/conversion/model/error/correspondence/DetailsErrorEquals.java

```diff
@@ -21,6 +21,9 @@ public boolean compare(Detail actual, LocalizedError expected) {
 			return expected == null;
 		}
 		ErrorCode error = actual.getErrorCode() == null ? null : ErrorCode.getByCode(actual.getErrorCode());
+
+
+
```

Comment:
These aren't needed, but whatever.

Reply:
Who is trying to pad their stats with blank lines??!!